css/New Evaluation.css

```diff
@@ -0,0 +1,46 @@
+body{
```

Comment:
I personally like to add a bit of spacing between selectors and relevant rulesets (i.e., before the `{`), as well as within rules, between the rule property and its corresponding value (i.e., after `:`). We'll want to standardize on the formatting at some point, to have things nice, clean and consistent.